simtools/configuration/commandline_parser.py

```diff
@@ -105,6 +105,12 @@ def initialize_path_arguments(self):
             default="./",
             required=False,
         )
+        _job_group.add_argument(
+            "--plain_output_path",
```

Comment:
I was actually expecting that we would replace the other `output_path` by the `plain_output_path` instead of adding a flag. But I am also fine with this. however, I would make clearer in the parameter name that this is a flag, e.g., `output_is_plain`

Reply:
`output_path`  without `plain_output_path` is still setting a path, including the tools and dates. I see the issue with the parameter name and renamed it to `use_plain_output_path`.